Visual_Studio_2017/GraphicalDebugging/ExpressionLoader.cs

```diff
@@ -210,13 +210,26 @@ public class MultiPointKindConstraint : KindConstraint
         private static MultiPointKindConstraint onlyMultiPoints = new MultiPointKindConstraint();
         public static MultiPointKindConstraint OnlyMultiPoints { get { return onlyMultiPoints; } }
 
+        /// <summary>
+        /// Loads object into watch to visualize.
```

Comment:
Actually it loads object and additional information into `IDrawable` and `Traits` which then can be used to visualize the variable in watch window.